components/script/dom/reportbody.rs

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+use dom_struct::dom_struct;
+
+use crate::dom::bindings::codegen::Bindings::ReportBodyBinding::ReportBodyMethods;
+use crate::dom::bindings::reflector::Reflector;
+
+#[dom_struct]
+pub(crate) struct ReportBody {
```

Comment:
Please add a link to https://w3c.github.io/reporting/#reportbody here, so you don't have to go to the idl file to lookup the specification.